problems/20.Count Days Spent Together.gs

```diff
@@ -1,47 +1,47 @@
 function countDaysTogether(arriveAlice: string, leaveAlice: string, arriveBob: string, leaveBob: string): number {
 
-    let Ama = parseInt(arriveAlice.substring(0, 2))
-    let Ada = parseInt(arriveAlice.substring(3, 5))
-    let Aml = parseInt(leaveAlice.substring(0, 2))
-    let Adl = parseInt(leaveAlice.substring(3, 5))
-    let Aaday: number = danUgodini(Ama, Ada)
-    let Alday: number = danUgodini(Aml, Adl)
-    let Adays: number[] = []
+    let aliMonArr = parseInt(arriveAlice.substring(0, 2))
```

Comment:
parseInt(arriveAlice.substring(0, 2)) 
It would be nicer if this was also a function correct? insated of seeing this magic numbers all over the place. If you wrap it in function and give it meaningful name 0, 2 are getting some meaning and like this they are just magic?